sedbackend/apps/core/files/storage.py

```diff
@@ -41,6 +42,18 @@ def db_save_file(con: PooledMySQLConnection, file: models.StoredFilePost) -> mod
 
 
 def db_delete_file(con: PooledMySQLConnection, file_id: int, current_user_id: int) -> bool:
+    stored_file_path = impl.impl_get_file_path(file_id, current_user_id)
+    
+    try:
+        os.remove(stored_file_path.path)
```

Comment:
We should assert that the path leads to, or is in a subdir of, FILE_RELATIVE_UPLOAD_DIR (defined on row 13), and throw an error if it does not, to prevent malicious attempts

Reply:
Should be resolved now.

Reply:
No, I decided against using pathlib. Have removed it now